tests/pkg-spine-order-svg/OPS/nav.xhtml

```diff
@@ -0,0 +1,12 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>SVG Spine Items in Order</title>
+  </head>
+  <body>
+    <nav epub:type="toc">
+      <ol>
+        <li><a href="1.svg">Link to first page</a></li>
```

Comment:
Consider linking to all 4.

Reply:
I'm a little wary to do this as the whole point of the test is to get to pages 2, 3, and 4 via the reading system user interface rather than directly via a link.

Reply:
Yeah, makes sense!